lib/tasks/gather.js

```diff
@@ -1091,6 +1103,12 @@ class TaskGather extends SttTask {
           this.emit('stt-error', evt);
           returnedVerbs = await this.performAction({reason: 'error', details: evt.error});
         }
+      } else if (reason.startsWith('stt-min-confidence-error')) {
+        if (this.parentTask) this.parentTask.emit('stt-min-confidence-error', evt);
+        else {
+          this.emit('stt-min-confidence-error', evt);
+          returnedVerbs = await this.performAction({reason: 'error', details: evt.error});
```

Comment:
I do not think we should be sending a webhook with reason 'error'.  This is not an error -- errors are when things fail, this is simply a case of not getting very recognizable audio.